blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpHandlerIntegrationTest.java

```diff
@@ -156,7 +113,14 @@ public void testHttpAnnotationsIngestionHappyCase() throws Exception {
         final int batchSize = 1;
         final String tenant_id = "333333";
         String event = createTestEvent(batchSize);
-        postEvent( event, tenant_id );
+        HttpResponse response = postEvent( tenant_id, event );
+
+        try {
+            assertEquals( 200, response.getStatusLine().getStatusCode() );
```

Comment:
Having only assert statement inside a try block seems odd. Cant you push the actual postEvent inside try block?